### PR TITLE
Update Hunt T1004 for #216, Adding Monitor for T1015

### PR DESCRIPTION
**Pull request by ION28**

### CODE REVIEW

**Comment by ION28** [comment]
Can use like `reg add "HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\sethc.exe" /v Debugger /t REG_SZ /d "c:\windows\system32\cmd.exe"` the following to test monitoring for T1015. Doesn't seem to be working for me

`.\BLUESPAWN-client.exe --monitor --log=console,debug --reaction=log --debug`

File: BLUESPAWN-common/headers/common/wrappers.hpp [hunk 1]
```diff
@@ -10,54 +10,19 @@
...
-	GenericWrapper& operator=(const GenericWrapper& copy){ 
-		freeResource = copy.freeResource; 
-		DestroyReference();
-		SetReference(copy.WrappedObject); 
-		return *this;
-	}
-	GenericWrapper&& operator=(GenericWrapper&& move) = delete;
-
-	~GenericWrapper(){ DestroyReference(); }
+	GenericWrapper(T object, std::function<void(T)> freeFunction = [](T object){ delete object; }, T BadValue = nullptr) :
```
Comment (ION28) [inline BLUESPAWN-common/headers/common/wrappers.hpp]: Codacy Issue found: [Class 'GenericWrapper < HandleWrapper >' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5033193)

Comment (Jack-McDowell) [reply BLUESPAWN-common/headers/common/wrappers.hpp]: Can we mark this as deliberate?

**Comment by ION28** [comment]
not prompting for reactions properly in monitor mode, but everything else seems to be working. Need to also test XML sink during monitor mode?

**Pull request merged**